talentprotocol/web-dapp
# feat: show pending approval talents for moderators

## Summary

This PR shows pending approval talents for moderators.

<img width="315" alt="Screenshot 2022-11-06 at 12 41 16" src="https://user-images.githubusercontent.com/33064059/200171334-17bf7974-e274-4b1b-827d-685660a72a1c.png">

## Notion link

https://talentprotocol.notion.site/Develop-Moderator-Role-2f254e51076c4910b2f086319a4cd161

## How to test?

- Sign in as a user with a moderator role
- Go to the talents page and there should be a "Pending approval" filter

## Notes

Following up on this PR > https://github.com/talentprotocol/web-dapp/pull/771
